examples/with-react-helmet/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "with-react-helmet",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "^2.0.0-beta",
+    "react": "^15.4.2",
+    "react-bootstrap": "^0.30.7",
+    "react-dom": "^15.4.2",
+    "react-helmet": "^4.0.0"
+  }
+}
```

Comment:
Please add author license and version to be consistent with https://github.com/zeit/next.js/blob/master/examples/basic-css/package.json#L14-L15